examples/stable-diffusion/run_8x.sh

```diff
@@ -0,0 +1,23 @@
+python ../gaudi_spawn.py --world_size 8 --use_mpi train_text_to_image_sdxl.py \
+  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution 512 \
+  --center_crop \
+  --random_flip \
+  --proportion_empty_prompts=0.2 \
+  --train_batch_size 16 \
```

Comment:
if you saw global step is 16*8,then you should change max_train_steps to 2500/8. Not sure if LR needs to be changed or not